readthedocs/rtd_tests/tests/test_core_utils.py

```diff
@@ -166,6 +166,7 @@ def test_trigger_build_invalid_time_limit(self, update_docs):
     def test_trigger_build_rounded_time_limit(self, update_docs):
         """Time limit should round down."""
         self.project.container_time_limit = 3
+        self.project.build_queue = 'build_queue'
```

Comment:
Not sure why we need this to make the tests pass. Where are we expecting this to be set?

Reply:
This was the last build that failed:
https://travis-ci.org/github/readthedocs/readthedocs.org/jobs/688452656

Reply:
There are `AssertionError`s near the bottom.

Reply:
For those cases, we just need to delete the expected value of queue `'queue': mock.ANY,`

Reply:
Will go ahead and remove 👍